doc/releasenotes.rst

```diff
@@ -8,6 +8,17 @@ Release Notes
 ..........
 
 
+0.9.15 (2019-09-15)
+-------------------
+
+* Some extensions of external simulators interface (#168).
+* Add basic plots of summary statistics (#165).
+* Document high-performance infrastructure usage (#159).
+* Some self-administratory stuff (#158, #157).
```

Comment:
what is this?

Reply:
that was basically about administration of the github repo (Add a social preview, add a link to zenodo). Didn't want to be too specific here since no user cares when these things were built in.